### ISSUE TITLE
Code does not parse thermochemistry data for centrosymmetric linear molecule

### ISSUE THREAD

**Issue by xinglong-zhang** [patonlab/GoodVibes#55]
Hi,
The codes does not seem to be able to obtain the thermochemical data for centrosymmetric linear molecules of D∞h point group. An example Gaussian output file is attached. This yields 

> x  ZnBr2_opt                                        ----   Caution! Potential invalid calculation of linear molecule from Gaussian
as the output.

Although there was a mention that the rotational temperature will be read directly by the program in linear molecules (https://github.com/patonlab/GoodVibes/pull/6), this does not seem to work in the current version. 

Any help to fix this please? Thank you.

[ZnBr2_opt.log](https://github.com/patonlab/GoodVibes/files/8674355/ZnBr2_opt.log)

**Comment by luchini18** [comment]
Hi Xinglong, 

Thanks for attaching the log file, we have issues when Gaussian itself doesn't identify the molecule as linear, as it will output an incorrect number of rotational constants and frequencies which can have an effect on the output thermochemistry.

We can usually get around this with the `symmetry=loose` argument, I've attached an output file below where I re-ran with this argument and GoodVibes is able to parse correctly (if you run with the --ssymm argument it will even identify the Dinfh point group)

Another quick way to check these is that we are expecting 3N-5 frequencies for linear molecules (4 freqs for this molecule), the initial frequency calculation only gave 3 vibrational modes and with the loose symmetry we were able to get 4 out.

Hope this helps!

[ZnBr2_opt_loose.log](https://github.com/patonlab/GoodVibes/files/8680955/ZnBr2_opt_loose.log)
